src/schema/search.ts

```diff
@@ -185,11 +170,6 @@ export const typeDefs = /* GraphQL */ `
       """
       query: String!
 
-      """
-      Version of the search algorithm
-      """
-      version: Int = 1
```

Comment:
Do we want to remove it? In the future we could have multiple search versions again and then we have to return all this code.

Reply:
Instead we could just adjust what default is (if not sent), eg. now to be `2`.

Reply:
Also open to that :) 
It might just be confusing what 2 means if there is no 1?

Reply:
I look at it as feed. There the old versions also phase out. So anything below current version will just get `2`.